README.md

```diff
@@ -1,3 +1,11 @@
+# To check for smi conformance of a deployed service mesh
+We use kuttl to check for SMI conformance. All the tests are writtten in smi-test directory of this repository.
+Execute the following command to run the smi-conformace tests:-
+
+```shell
+kubectl kuttl test  --skip-cluster-delete=true --start-kind=false ./smi-test
```

Comment:
Ultimately, a capability to expose through Meshery's API and then through `mesheryctl`?